src/components/DocumentationTopic.vue

```diff
@@ -354,6 +355,10 @@ export default {
       type: Boolean,
       default: false,
     },
+    standardColorIdentifier: {
+      type: String,
```

Comment:
Not a blocker but we should validate that the provided string is one of the fixed set of options.